### ISSUE TITLE
Cannot parse docx document containing more than one element

### ISSUE THREAD

**Issue by mmolenda** [chrismattmann/tika-python#62]
Hi, first of all thanks for the great tool.
When I parse a docx document that contains several components, for example document itself and an image, for subsequent ones the parser tries to append values to already existing keys in metadata dict. But in some cases the key contains string/unicode and append operation results in `AttributeError`.

Example:
```
In [1]: from tika import parser

In [2]: parsed = parser.from_file('https://dl.dropboxusercontent.com/u/3408905/197.docx')
tika.py: Retrieving https://dl.dropboxusercontent.com/u/3408905/197.docx to /var/folders/nb/y4j12k7d6tq3_vkx118_602w0000gn/T/197.docx.
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-2-0599c0dd20fe> in <module>()
----> 1 parsed = parser.from_file('https://dl.dropboxusercontent.com/u/3408905/197.docx')

/private/tmp/tikagit/tika-python/tika/parser.py in from_file(filename, serverEndpoint)
     23 def from_file(filename, serverEndpoint=ServerEndpoint):
     24     jsonOutput = parse1('all', filename, serverEndpoint)
---> 25     return _parse(jsonOutput)
     26
     27 def from_buffer(string, serverEndpoint=ServerEndpoint):

/private/tmp/tikagit/tika-python/tika/parser.py in _parse(jsonOutput)
     51             if n != "X-TIKA:content":
     52                 if n in parsed["metadata"]:
---> 53                     parsed["metadata"][n].append(js[n])
     54                 else:
     55                     parsed["metadata"][n] = js[n]

AttributeError: 'unicode' object has no attribute 'append'
```

I think if a same key is encountered second time, the existing value should be converted to a list (if it's not already), probably something like this should fix the issue:
```
diff --git a/tika/parser.py b/tika/parser.py
index c6ce684..9053bca 100644
--- a/tika/parser.py
+++ b/tika/parser.py
@@ -50,6 +50,8 @@ def _parse(jsonOutput):
         for n in js:
             if n != "X-TIKA:content":
                 if n in parsed["metadata"]:
+                    if not isinstance(parsed["metadata"][n], list):
+                        parsed["metadata"][n] = [parsed["metadata"][n]]
                     parsed["metadata"][n].append(js[n])
                 else:
                     parsed["metadata"][n] = js[n]
```

**Comment by chrismattmann** [comment]
Thanks @mmolenda  I will take a look and update this!

**Comment by chrismattmann** [comment]
this closes #62
